test/util/asserts.ts

```diff
@@ -2,6 +2,79 @@ import type { EventRecord } from "@polkadot/types/interfaces";
 import { strictEqual } from "node:assert";
 import type { ApiPromise } from "@polkadot/api";
 import type { AugmentedEvent } from "@polkadot/api/types";
+import type { BspNetApi } from "./bspNet";
+
+//TODO: add ability to search nested extrinsics e.g. sudo.sudo(balance.forceTransfer(...))
+export const assertExtrinsicPresent = async (
```

Comment:
Might be worth adding some doc comments on what the options are and what is their behaviour.